build/ProjectReunion-Versions.yml

```diff
@@ -0,0 +1,12 @@
+variables:
+  # Version Control
+  major: "0"
+  minor: "5"
+  patch: "0"
+  branch: 'main'
```

Comment:
the branch specific stuff needs to be in a separate file.

Reply:
I don't think I agree with that. This is all part of the version name so we should just keep it here instead of in a separate place

Reply:
if we change the format, we have to manually port the change to different branches (i.e. cannot cherrypick). Not a big deal though as this is a small file.

Reply:
Ah I see what you meaannn

Reply:
Sticking the version into its own specific file is ideal. Easiest to manage, as it's purely data (separate from tasks/jobs/script/etc which are 'code')

Reply:
I can move it to a specific yml file like this one. This yml file is meant to be purely data.
I can totally separate the variables other than major, minor, and patch to another file called ProjectReunion-BuildData.yml or something of sort.